### PR TITLE
Dpp 1669 transaction type 16

### PR DESCRIPTION
**Pull request by carlbm**

### CODE REVIEW

File: src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Specs/WhenANewEventIsGenerated.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void ThenItShouldBeWrittenToTheDatabase(TestFixtureContext context)
             Assert.IsNotNull(eventCommitmentVersions);
 
             Assert.AreEqual(1, eventErrors.Length);
-            Assert.AreEqual(48, eventPeriods.Length);
+            Assert.AreEqual(60, eventPeriods.Length);
```
Comment (Pauljgraham) [inline src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Specs/WhenANewEventIsGenerated.cs:68]: just as a matter of interest why does increase to 60?

**Review by Pauljgraham — APPROVED** [review]
One tiny comment, where i didn't understand where 12 additional records would come from.

File: src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Specs/WhenANewEventIsGenerated.cs (hunk 1 above)
Comment (carlbm) [reply src/SharedPipelineComponents/DataLockEvents/SFA.DAS.Provider.Events.DataLock.IntegrationTests/Specs/WhenANewEventIsGenerated.cs:68]: The test is looking for the number of Datalock.PriceEpisodePeriodMatch rows. The sample data has all transaction types and is over 12 months, so will produce one row per month per TransactionTypeFlag (or better called census date type). We've moved from 4 to 5 census date types, so went from 4 X 12 = 48 to 5 X 12 = 60 rows in the table

**Comment by satya-sanikommu** [comment]
Review and Testing completed. Ready to merge

**Pull request merged**